RadixWallet/Clients/ROLAClient/ROLAClient+Live.swift

```diff
@@ -10,16 +10,17 @@ extension ROLAClient {
 			let entity = request.entity
 			let newPublicKey = request.newPublicKey
 
-			let entityAddress: Address = switch entity {
+			let entityAddress = switch entity {
 			case let .account(account):
-				account.address.asGeneral
+				AddressOfAccountOrPersona.account(account.address)
```

Comment:
Quite a few API's work with **either** Account **or** Identity (Persona) Address, so will be useful to have a tagged union for it.